ios/Classes/SwiftAppTrackingTransparencyPlugin.swift

```diff
@@ -33,17 +46,16 @@ public class SwiftAppTrackingTransparencyPlugin: NSObject, FlutterPlugin {
     case authorized = 3
   */
   private func requestTrackingAuthorization(result: @escaping FlutterResult) {
-        
     if #available(iOS 14, *) {
-         #if canImport(AppTrackingTransparency)
+        #if canImport(AppTrackingTransparency)
         ATTrackingManager.requestTrackingAuthorization(completionHandler: { status in
 
             switch status {
```

Comment:
I want to remove these debugging print statements. Since we return the status value, developers can print it themselves if they want.

Reply:
You are right. `#if canImport(AppTrackingTransparency)` statements are redundant now. We also clearly state that this plugin requires xcode 12 and above in notes section of readme.md. Its safe to remove those checks.